include/base/users/class-bd-user-meta-box-module.php

```diff
@@ -206,13 +208,21 @@ protected function render_user_login_restrict_settings() {
 	 * @since 5.5
 	 */
 	protected function render_user_with_no_posts_settings() {
-?>
+	?>
 		<tr>
 			<td scope="row" colspan="2">
 				<input name="smbd_<?php echo $this->field_slug; ?>_no_posts" id="smbd_<?php echo $this->field_slug; ?>_no_posts" value="true" type="checkbox">
 				<?php _e( "Only if user doesn't have any post. Posts from 'post', 'page' & all custom post types would be considered.", 'bulk-delete' ); ?>
 			</td>
 		</tr>
-<?php
+	<?php
+		if ( function_exists( 'bd_render_post_type_dropdown' ) ) {
```

Comment:
You can remove the `function_exists` check here

Reply:
Fixed.